docs/platforms/python/integrations/bottle/index.mdx

```diff
@@ -73,22 +73,22 @@ sentry_sdk.init(
 
 ```
 
-You can pass the following keyword arguments to `BottleIntegration()`:
+Each of the following options are supported as keyword arguments to `BottleIntegration()`.
 
-- `transaction_style`:
+### `transaction_style`
 
-  ```python
-  @app.route("/myurl/<foo>")
-  def myendpoint():
-      return "ok"
-  ```
+The `transaction_style` option specifies how the transaction name is generated. You can set the option to `"endpoint"` (the default) or `"url"`.
 
-  In the above code, you would set the transaction to:
+For example, given the following route:
 
-  - `/myurl/<foo>` if you set `transaction_style="url"`.
-  - `myendpoint` if you set `transaction_style="endpoint"`
+```python
+@app.route("/myurl/<foo>")
+def myendpoint():
+    return "ok"
+```
 
-  The default is `"endpoint"`.
+- If you set `transaction_style="endpoint"`, the transaction name is `"myendpoint"`, since that is the route handler function's name.
```

Comment:
```suggestion
- If you set `transaction_style="endpoint"`, the transaction name will be `"myendpoint"`, since that is the route handler function's name.
```